src/app/verification-addresses/features/iva-manager-list/iva-manager-list.component.scss

```diff
@@ -0,0 +1,15 @@
+.stateUnverified {
+  color: #dc2626;
```

Comment:
We should use the colour vars defined by the material colour palette whenever possible.